modules/oneKeyIdSystem.md

```diff
@@ -0,0 +1,104 @@
+# OneKey
+
+The OneKey real-time data module in Prebid has been built so that publishers
+can quickly and easily setup the OneKey Addressability Framework.
+This module is used along with the oneKeyRtdProvider to pass OneKey data to your partners.
+Both modules are required. This module will pass oneKeyData to your partners
+while the oneKeyRtdProvider will pass the transmission requests.
+
+Background information:
+- [prebid/addressability-framework](https://github.com/prebid/addressability-framework)
```

Comment:
my understanding is this is not a stable location, perhaps delete these two lines or point to the openx fork or point to the moved repo?

Reply:
(Duplicated answer from https://github.com/prebid/Prebid.js/pull/8686#discussion_r940166567)

For now, there is no other location. As far as I know, nothing is planned for changing them. If later, those locations change, we will update this document accordingly.

Reply:
@RomainLofaso it seems we're approaching the finish line here, looking forward to merging once the branding changes are complete in this pr

Reply:
Hello @patmmccann ,

Sorry, I took time to do this change. Briefly, my team wasn't aware of this aspect of the vote. After your feedback, we clarified this point. Finally, @BasileLeparmentier has moved the repositories to another workspace so that I can update the PR with a good location.

This is done in the last commit.

Thanks